test/functional/rpc_fundrawtransaction.py

```diff
@@ -969,6 +970,44 @@ def test_include_unsafe(self):
         signedtx = wallet.signrawtransactionwithwallet(fundedtx['hex'])
         wallet.sendrawtransaction(signedtx['hex'])
 
+    def test_22670(self):
+        # In issue #22670, it was observed that ApproximateBestSubset may
+        # choose enough value to cover the target amount but not enough to cover the transaction fees.
+        # This leads to a transaction whose actual transaction feerate is lower than expected.
+        # However at normal feerates, the difference between the effective value and the real value
+        # that this bug is not detected. It is most easily observed when the minimum relay fee is set to
+        # an extremely high value (0.1 BTC/kvB in this test).
+        self.log.info("Test issue 22670 ApproximateBestSubset bug")
+        # Make sure the default wallet will not be loaded when restarted with a high minrelaytxfee
+        self.nodes[0].unloadwallet(self.default_wallet_name, False)
+        self.restart_node(0, ["-minrelaytxfee=0.1", "-changetype=bech32"])
```

Comment:
Maybe create `DEFAULT_MAX_RAW_TX_FEE_RATE` constant and use that so you can refer to it later as well?

Reply:
Done.. ish